constants.py

```diff
@@ -2,7 +2,7 @@
 from datetime import datetime, timedelta
 import requests
 
-DAYS = 30
+DAYS = (30,)
```

Comment:
Are you sure that DAYS should be a tuple like this? Should it not be an integer `DAYS = 30`?